src/cogs/help.py

```diff
@@ -0,0 +1,234 @@
+"""
+Mainly a help command that replaces the default command that discord.py provides.
+This module also contains related commands and functions such as about, contrib, docs.
+"""
+import configparser
+from contextlib import suppress
+
+import discord
+from discord.ext import commands
+
+
+def find_ci(dictionary, string):
+    """
+    Returns a value for a key in a dictionary. Unlike normal indexing
+    this is case insensitive, and returns None if no value exists
+    """
+    for key, value in dictionary.items():
+        if key.lower() == string.lower():
+            return value
+    return None
+
+
+def cog_values(self):
```

Comment:
It's unclear what "self" is here, since self isn't part of a class. I'd either move it into the Help cog, or change the name of it's argument. Personally, instead of passing in the "Help" cog, I'd pass in the discord bot instead.